src/commands/config/envs.js

```diff
@@ -3,14 +3,15 @@ const path = require('path');
 
 const { flags: flagTypes } = require('@oclif/command');
 
+const { HOME_DIR_PATH: homeDirPath } = require('../../constants');
```

Comment:
I would use the constant name to keep more visible when you read the code.

Reply:
OK, I changed all of the destructuring to just use the constant instead, does [this](https://github.com/dashevo/dashmate/pull/437/commits/e8b83f81283e6966685d52c5a9464ec39578898a) look right? Or should it only be in `envs.js`?